changelog/1.1.0.md

```diff
@@ -10,12 +10,15 @@
 * Added support for providing ranges as length two tuples to `addLogisticRegression` and `addRandomForest`.
 * Added new method `auto_tuning_config` to NC and LP pipelines for querying a pipelines auto-tuning config.
 * Added support for new GDS library 2.1 signature of gds.graph.removeNodeProperties.
+* Added new function `gds.alpha.graph.construct` to costruct a GDS graph from pandas `DataFrame`s, which works if the GDS Flight server is enabled.
```

Comment:
```suggestion
* Added new function `gds.alpha.graph.construct` to construct a GDS graph from pandas `DataFrame`s, which works if the GDS Flight server is enabled.
```

Reply:
Also, two lines above, you say `method` instead of `function`. not sure if this is on purpose :shrug:

Reply:
I noticed this too, but personally I don't care - both statements are true. Do you want us to change it?

Reply:
change to `method` now

Reply:
done